### ISSUE TITLE
Azure Function App Debugging Issue

### ISSUE THREAD

**Issue by ZanattaMichael** [PowerShell/vscode-powershell#2925]
### System Details

System Details Output
```
### VSCode version: 1.48.2 a0479759d6e9ea56afa657e454193f72aef85bd0 x64

### VSCode extensions:
aaron-bond.better-comments@2.1.0
cliffordfajardo.hightlight-selections-vscode@0.0.1
DavidAnson.vscode-markdownlint@0.36.3
eamodio.gitlens@10.2.2
evrowe.cold-snack@1.6.0
fireside21.cshtml@0.1.3
formulahendry.github-actions@0.0.1
johnpapa.vscode-peacock@3.8.0
karigari.chat@0.35.0
luggage66.VBScript@0.0.2
ms-azuretools.vscode-azureappservice@0.18.0
ms-azuretools.vscode-azurefunctions@0.24.0
ms-azuretools.vscode-azurestorage@0.9.0
ms-dotnettools.csharp@1.23.1
ms-python.anaconda-extension-pack@1.0.1
ms-python.python@2020.8.105045
ms-vscode-remote.remote-containers@0.134.1
ms-vscode-remote.remote-wsl@0.44.4
ms-vscode.azure-account@0.9.2
ms-vscode.powershell@2020.4.0
ms-vsliveshare.vsliveshare@1.0.2731
ms-vsliveshare.vsliveshare-audio@0.1.85
ms-vsliveshare.vsliveshare-pack@0.4.0
Nuuf.theme-hackershaze@2.0.1
quicktype.quicktype@12.0.46
redhat.vscode-yaml@0.10.1
rosshamish.kuskus-kusto-syntax-highlighting@1.1.8
streetsidesoftware.code-spell-checker@1.9.0
Yummygum.city-lights-icon-vsc@1.1.3
yummygum.city-lights-theme@1.1.8
yycalm.linecount@0.1.7


### PSES version: 2.1.0.0

### PowerShell version:

Name                           Value
----                           -----
PSVersion                      7.0.3
PSEdition                      Core
GitCommitId                    7.0.3
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

```

### Issue Description

Debugging within Function App not working as expected. Can enter debug session using Wait-Debugger. You can step through the code, however you cannot execute powershell statements from the console.

#### Expected Behavior

While the debugger is attached the console should be able to execute PowerShell statements.

#### Actual Behaviour

Upon inputting a statement, the console doesn't respond.

### Attached Logs

Follow the instructions in the [troubleshooting docs](https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md#logs)
about capturing and sending logs.

**Comment by TylerLeonhardt** [comment]
@ZanattaMichael are you attaching to a PSCore 6 Function App or a PS7 Function App? Note that the version of the Function App can be different than the version of PowerShell you have installed on your machine.

**Comment by ZanattaMichael** [comment]
@TylerLeonhardt. Sorry PS7. The issue is resolved if I roll back to v2020.4.0.

**Comment by SydneyhSmith** [comment]
Thanks @ZanattaMichael it would be really helpful if you could also attach the logs--instructions for how to do that are [here ](https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md#providing-information-about-your-environment)
